README.md

```diff
@@ -153,7 +153,7 @@ user.getCards().then(new PromiseAction<List<Card>>() {
 CardRequest cardRequest = new CardRequest("label", "USD");
 user.createCard(cardRequest);
 
-// Or a card request with the label, currency, position and whether it is starred or not.
+// Or a card request with the label, currency, position and whether it is started or not.
```

Comment:
Please undo this change.

Reply:
Starred is not a typo, it refers to an aspect of the functionality of the UI. If a user has placed a star on the card it is starred, meaning that it has been favorited.